spython/tests/test_oci.py

```diff
@@ -61,34 +62,53 @@ def test_oci(self):
         print(result)
         self.assertEqual(result['status'], 'created')
 
-        print('...Case 3. Execute command to running bundle.')
+        print('...Case 3. Execute command to non running bundle.')
         result = self.cli.oci.execute(container_id=self.name, 
-                                      sudo=True, 
-                                      command=['ls','/'])
+                                      sudo=True,
+                                      command=['ls', '/'])
 
         print(result)
-        self.assertTrue('bin' in result)
+        print(self.cli.version_info())
 
-        print('...Case 4. Check status of existing bundle.')
-        state = self.cli.oci.state(self.name, sudo=True)
-        self.assertEqual(state['status'], 'created')
+        if self.cli.version_info() >= VersionInfo(3, 2, 0):
```

Comment:
IMO this isn't correct. Can you setup a test job with 3.2.0? I think the change was to 3.2.1 so this may fail

Reply:
Sure I’ll add to the other PR. If that doesn’t work do you know how to account for the last 1 in 3.3.1-1? The original check you had for >= 3.2.1 returned False.